Jenkinsfile

```diff
@@ -534,6 +534,24 @@ pipeline {
                     }
                 }
 
+                stage('Hip Implicitgemm layouts Release') {
+                    agent{ label rocmnode("gfx908") }
+                    environment{
+                        cmd = '''
+                            ulimit -c unlimited
+                            rm -rf build
+                            mkdir build
+                            cd build
+                            CXX=/opt/rocm/llvm/bin/clang++ cmake -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=release -DMIOPEN_GPU_SYNC=On .. 
+                            make -j test_conv_for_implicit_gemm_mlir
```

Comment:
This only builds the test, but does not run it. Fixed.

Reply:
This `test_conv_for_implicit_gemm_mlir` used to be a `add_custom_test` target so it will actually run the test, but only on the small subset. 

Now I can see what you did here is to enable both `MIOPEN_USE_MLIR` and `MIOPEN_TEST_MLIR` in the jenkins stage and test out the entire ctest suite, which does give more clarity on that stage. 

I guess the long term goal, when MLIR path is stable, is to flip the two cmake variable to make them default on?

Reply:
MLIR may affect other tests. Let's run ALL the tests with MLIR. Resolved.

Reply:
> I guess the long term goal, when MLIR path is stable, is to flip the two cmake variable to make them default on?

We shall test the configuration(s) that is delivered to the end users. As soon as MLIR becomes the default, it will be tested everywhere, so we'll remove "special" test stage for MLIR.

Reply:
> MLIR may affect other tests. Let's run ALL the tests with MLIR. 
As soon as MLIR becomes the default, it will be tested everywhere, so we'll remove "special" test stage for MLIR.

Yes, sounds reasonable.